redirects.csv

```diff
@@ -53,7 +53,6 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /profile/personal,/profile/personal-info,302!
 /profile/*,https://${env:CRDS_PROFILE_ENDPOINT}/:splat,200! Role=user
 /profile/*,/signin?redirectUrl=/profile/:splat,302!
-/signin,https://${env:CRDS_ANGULAR_ENDPOINT}/signin,200!
```

Comment:
Should this go to prod I think Devy may be working on testing Okta ([Slack Message](https://crdschurch.slack.com/archives/C02C3F925/p1608231715112100)).

Reply:
This should not go into prod